src/renderer/i18n/de/wallet.ts

```diff
@@ -44,6 +44,7 @@ const wallet: WalletMessages = {
   'wallet.create.words.click': 'Klicke die Wörter in der richtigen Reihenfolge',
   'wallet.create.creating': 'Erstelle eine Wallet ...',
   'wallet.create.error': 'Fehler beim Abspeichern der Phrase',
+  'wallet.create.error.phrase': 'Save your phrase safely and enter it in a correct way - DE',
```

Comment:
'wallet.create.error.phrase': 'Speicher Deine Phrase an einem sicherem Ort und gebe diese in der richtigen Reihenfolge ein',